include/relation/schema.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+#include "relation/value/value.h"
+
+namespace query_process_engine {
+
+struct SchemaItem {
+  std::string relation_;
+  std::string attribute_;
+  ValueType type_;
+};
+class Schema {
+ private:
+  std::vector<SchemaItem> schema_;
```

Comment:
current schema class is will reorder schema item when return, which is unexpected sometimes.

Reply:
Seems it's only about db schema in mock transactional storage manager. 

It DO NOT affect other parts actually, now the method `get_relation_schema` will sort schema by insert sequence .

Just change implementations in mock storage manager, this schema class works fine in current process.

Reply:
In preprocessor, our schema items are inserted group by relations, thus sort by position is enough to get compatible with down-side schema.

Retain current schema, though it is really strange.